integ-test/build.gradle

```diff
@@ -37,6 +37,7 @@ repositories {
 
     maven {
         url  "http://dl.bintray.com/steppschuh/Markdown-Generator"
+        allowInsecureProtocol = true
```

Comment:
Never use insecure! Change the URL to https (ideally replace with a library from Maven Central)